requirements.txt

```diff
@@ -1,2 +1,6 @@
 qiskit==0.26.2
 cirq==0.11.0
+numpy==1.20.1
```

Comment:
@abhimanyukumbhar Pls remove numpy as it is taken care of by qiskit

Reply:
Okay sir!